src/utils/pause/mapScopes/findGeneratedBindingFromPosition.js

```diff
@@ -95,15 +135,43 @@ async function findGeneratedReference(
     end: Location
   }
 ): Promise<GeneratedDescriptor | null> {
-  return generatedAstBindings.reduce(async (acc, val) => {
+  const bindings = filterApplicableBindings(generatedAstBindings, mapped);
```

Comment:
It might even be nice to bail here of we get "too many" possible bindings, but I opted to leave that out for now.